core/model/pom.xml

```diff
@@ -37,6 +37,11 @@
             <artifactId>commons-lang3</artifactId>
```

Comment:
Why not drop the deprecated dependency?

Reply:
Because not the whole package is deprecated; it's still valid e.g. for `StringUtils`, but `StringEscapeUtils` was indeed moved.